Description: Fixed hardening format security warning.
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2012-09-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/asn1c/unber.c
+++ b/asn1c/unber.c
@@ -392,7 +392,7 @@
 		return;
 	}
 
-	while(level-- > 0) printf(indent_bytes);  /* Print indent */
+	while(level-- > 0) printf("%s", indent_bytes);  /* Print indent */
 	printf(fin ? "</" : "<");
 
 	printf(constr ? ((tlv_len == -1) ? "I" : "C") : "P");
